fiftyone/core/odm/database.py

```diff
@@ -135,26 +135,23 @@ def get_db_config():
     return config
 
 
-def cleanup_multiple_config_docs(conn, config_docs):
-    """Internal utility that ensures that there is only one
-    :class:`DatabaseConfigDocument` in the database.
-    """
+def _handle_multiple_config_docs(conn, config_docs):
+    # Use the newest one
+    keep_doc = max(config_docs, key=lambda d: d["_id"])
```

Comment:
I think we should not do this anymore. This function should accept `conn` only. Then re-query for the config document that survived at the end. Avoids another potential concurrency issue

Reply:
Updated 👍